src/wp-includes/class-wp-block-processor.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+/**
+ * @todo Benchmark extracting blocks against {@link \parse_blocks()}.
+ */
+class WP_Block_Processor extends WP_Block_Scanner {
+	private $open_blocks = array();
+
+	private $was_void = false;
+
+	public static function create( $html ) {
+		return new self( $html );
+	}
+
+	public function next_delimiter( $block_type = null, $html_spans = 'skip-html' ) {
```

Comment:
The params in this method are different than those in `WP_Block_Scanner`.

Reply:
this class is gone now, so this is no longer relevant.